src/features/slackapi/slackapi-utils.ts

```diff
@@ -205,12 +218,20 @@ Have a great week!
       const firstDayBack = TimeUtilities.checkIfUserIsAwayOrIsItFirstDayBack(timeRegistrations, personId, expected, yesterday, nonProjectTimes);
 
       const message = constructWeeklySummaryMessage(userData, weekStartDate.toISODate(), weekEndDate.toISODate());
-
       if (!isAway && !firstDayBack) {
```

Comment:
The removed blank line above could be added back in. Typically we have a blank line between any variables and a conditional statement